tools/evm/T8NTool/T8NToolTracer.cs

```diff
@@ -0,0 +1,242 @@
+﻿using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Evm;
+using Nethermind.Evm.Tracing;
+using Nethermind.Int256;
+
+namespace Evm.T8NTool
+{
+    public class T8NToolTracer : IBlockTracer, ITxTracer, IJournal<int>, ITxTracerWrapper
```

Comment:
did you check if GethLikeTxMemoryTracer and GethLikeBlockMemoryTracer aren't suitable for this?